pkgs/development/python-modules/pgcli/default.nix

```diff
@@ -8,6 +8,7 @@
   configobj,
   prompt-toolkit,
   psycopg,
+  python3,
```

Comment:
We cannot use python3 in python-modules as that would mix python versions and break overlays.